### DISCUSSION TITLE
Cookie control with nuxt 2

### DISCUSSION THREAD

**Discussion by VentsislavB** [dargmuesli/nuxt-cookie-control#93] [Q&A] [answered]
Hello,

I am trying to use this library for my nuxt 2 project but the autoimporting seems not to be working. Is there a way to use the component without autoimporting or how can I use this library with nuxt 2?

Thank you in advance!

**Comment by dargmuesli** [comment, marked as answer]
This repository does not support Nuxt 3 any more. For Nuxt 2 use https://gitlab.com/broj42/nuxt-cookie-control!
